File: python/Easy/242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        l=[0]*26
        d = l.copy()
        for i in s:
            l[ord(i)-ord('a')]+=1
        for i in t:
            l[ord(i)-ord('a')]-=1
        return l==d

File: c++/easy/242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {

        if (s.size() != t.size()) {
            return false;
        }
        std::vector<int> alpha(26, 0);
        for (const char& c : s) {
            int index = c - 'a';
            alpha[index]++;
        }
        for (const char& c : t) {
            int index = c - 'a';
            alpha[index]--;
            if (alpha[index] < 0) {
                return false;
            }
        }
        return true;
    }
};